docs/testplan/pfc/PFC_PAUSE_RESPONSE_HEADROOM_README.md

```diff
@@ -0,0 +1,48 @@
+This document describes how to test the effect of PFC response times (a delay in response to pfc frames) from the IXIA on the headroom of the SONiC DUT for lossless traffic. For the purposes of our test, the data has been collected for the Cisco 8102 and Arista 7050CX3 switches. 
+
+## Background
+PFC pause frames can prevent a switch queue from sending any data in a time interval. The PFC frame (Figure 1 in https://www.cisco.com/c/en/us/products/collateral/switches/nexus-7000-series-switches/white_paper_c11-542809.html) has a 8-bit field called 'cass-enable vector' where the bit N tells if the frame pauses priority N. The bit value of 1 means pause. For a priority to pause, the PFC frame also needs to specify its pause duration. The PFC frame has 8 pause durations fields, one for each priority. The pause duration for each priority is a 2-byte value that expresses time as a number of quanta, where each quanta represents the time needed to transmit 512 bits at the current network speed. For example, the value of 65535 at a 40 Gbps link means 65535 * 512bits / 40Gbps = 838.84 microseconds. Therefore, to fully block a switch queue, we need to generate PFC pause frames fast enough. For example, to block 40G link, we need to generate a PFC pause frame with 65535 quantas in every 838.84 microseconds. A pause duration of zero quanta has the special meaning of unpausing a priority.
+
+SONiC has two lossless priorities: 3 and 4, by default. Note that only the lossless priorities can react to or generate PFC frames. In other words, PFC frames should not have any impact on traffic on lossy priorities. Packets with Differentiated Services Code Point (DSCP) 3 and 4 are mapped to priority 3 and 4, respectively.
+
+### PFC Headroom
+There is a time delay between when the sender of the PFC pause frame sends the PFC Pause Frame, and the receiver begins to pause traffic flow. This PFC delay requires the PFC Pause Frame sender (the switch) to reserve some buffer space to absorb in-flight packets sent during this time. This reserved buffer space is called "PFC headroom". The PFC Headroom value depends on several factors, including the response time of the sender to react to PFC Pause frames upon receiving one - known as the PFC response time. 
```

Comment:
Do not use "PFC delay" here. The reader may think it is "PFC response time". Just say

After a device sends a PFC pause frame, it will take some time for the pause frame to arrive at the upstream device and take effect. To absorb packets arriving during this period of time, we need to reserve PFC headroom ..